File: 3581-TheTwoSneakyNumbersOfDigitville/3581-TheTwoSneakyNumbersOfDigitville.js
// Last updated: 3/7/2026, 10:01:55 PM
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    let arr = [];
    let count = 0;

    for(let i=0; i<nums.length; i++){
        count = 0;
        for(let j=i; j<nums.length; j++){
            if(nums[i] == nums[j]){
                count ++
            }
        }
        if(count == 2){
            arr.push(nums[i])
        }
        if(arr.length == 2){
            return arr;
            break;
        }
    }
    return arr;
};

File: 3581-TheTwoSneakyNumbersOfDigitville/3581-TheTwoSneakyNumbersOfDigitville.cpp
// Last updated: 8/13/2026, 10:37:23 AM
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        unordered_map<int, int> freq;
        vector<int> res;

        for(int i=0; i<nums.size(); i++){
            freq[nums[i]]++;
        }

        for(auto it : freq){
            if(it.second == 2){
                res.push_back(it.first);
            }
        }
        return res;
    }
};